handbook/product/README.md

```diff
@@ -209,6 +209,25 @@ When writing copy, consider whether additional information is necessary before a
 
 When writing the time in the UI using "am" and "pm" abbreviations, write them **without space** between time and abbreviation, with **no punctuation**, and use **lowercase** letters (e.g. Working hours are 8am to 5pm).
 
+**Writing the error messages**
+
+When writing error messages in the UI or CLI, follow these rules:
+- Write the **error** and **possible solution** (e.g. No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.), or
```

Comment:
At first glance, I wasn't sure which rule I should use and when. To solve this, I suggest starting each of these bullets with an explanation on why I should this rule. 

For example, we could update the first bullet to something like this:

`If the solution to the error isn't obvious, write a message with the **error** followed by the **solution**. For example, "No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.)"`

Also smaller point, should we just say "solution" instead of "possible solution"? Possible solution to me reads like it might fix the issue or it might not.